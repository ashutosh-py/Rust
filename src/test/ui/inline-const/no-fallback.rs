// test for #78132, see the comment in `fn typeck` for why this
// is supposed to fail.
#![feature(inline_const)]
#![allow(incomplete_features)]

fn main() {
    let _: usize = const { 0 };
    //~^ ERROR type annotations needed
}
