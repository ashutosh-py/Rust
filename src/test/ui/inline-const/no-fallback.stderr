error[E0282]: type annotations needed
  --> $DIR/no-fallback.rs:7:26
   |
LL |     let _: usize = const { 0 };
   |                          ^^^^^ cannot infer type for type `{integer}`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0282`.
